src/axom/slic/core/Logger.cpp

```diff
@@ -16,8 +16,16 @@ namespace axom
 {
 namespace slic
 {
-Logger* Logger::s_Logger = nullptr;
-std::map<std::string, Logger*> Logger::s_loggers;
+static Logger* s_Logger = nullptr;
```

Comment:
I'm not sure that `s_Logger` is just a cache though, since it's used in `slic::isInitialized()` and `Logger::getActiveLogger()`. Please see my overall PR comment.